pkg/validation/limits.go

```diff
@@ -277,6 +278,9 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	_ = l.MaxCacheFreshness.Set("10m")
 	f.Var(&l.MaxCacheFreshness, "frontend.max-cache-freshness", "Most recent allowed cacheable result per-tenant, to prevent caching very recent results that might still be in flux.")
 
+	_ = l.MaxMetadataCacheFreshness.Set("24h")
```

Comment:
I think it's worth explaining the 24h here, since this aligns with the index compaction period & the index query splitting period. These are all related but since we have no relationship established it will be very tricky to know if we have modified all the correct places if we ever change these periods.